### ISSUE TITLE
Wacom Support Flutter SDK

### ISSUE THREAD

**Issue by ervijjusty** [woodemi/notepad_core#123]
When the wacom supports is available for use for flutter sdk?

**Comment by Sunbreak** [comment]
> When the wacom supports is available for use for flutter sdk?

We just reverse engineer the Wacom protocol and have a kotlin version. Will you help to port to Flutter?

**Comment by vincenttarrit** [comment]
Wacom has a sdk for android and ios, do you think we could use this to implement this plugin ?

**Comment by Sunbreak** [comment]
> Wacom has a sdk for android and ios, do you think we could use this to implement this plugin ?

IIRC, it is an SDK for Ink data parse and store

**Comment by vincenttarrit** [comment]
yes right and a little bit more i think. Do you think I could see your work in kotlin to move it in Swift ?

**Comment by Sunbreak** [comment]
> yes right and a little bit more i think. Do you think I could see your work in kotlin to move it in Swift ?

Move `kotlin` to `Swift`? This is a `Flutter` plugin, not `Kotlin/MPP`

**Comment by vincenttarrit** [comment]
yes sure, i just want to help to produce something in flutter

**Comment by Sunbreak** [comment]
> yes sure, i just want to help to produce something in flutter

Do you mean `open source the Kotlin/MPP implementation of  Wacom Support` and you'd like to port it into Flutter plugin?

**Comment by vincenttarrit** [comment]
yes i could try

**Comment by Sunbreak** [comment]
[library_src_main_java_io_woodemi_notepad_wacom.zip](https://github.com/woodemi/notepad_core/files/5539830/library_src_main_java_io_woodemi_notepad_wacom.zip)

Here are `WacomClient.kt` & `WacomCommand.kt` which are to be added into https://gitlab.com/woodemi/sdk/notepad-sdk-mpp but not yet

**Comment by vincenttarrit** [comment]
great thank you, i tried and got some stuff, but i have some question about it, can you contact me on telegram ? thank you !

**Comment by Sunbreak** [comment]
There's some limit to my telegram account. What about discord? There are all the Flutter teams

**Comment by vincenttarrit** [comment]
Yes great my id is Vincent#8004

**Comment by Sunbreak** [comment]
Mine is sunbreak#7274. Sent a request
